src/frontend/web/src/ratingpage/RatingCard.css

```diff
@@ -0,0 +1,66 @@
+.rating-card {
+  text-align: center;
+  width: 55vh;
+  height: 74vh;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 5vh;
+  margin-bottom: 5vh;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.rating-card > .ant-card-body {
+  padding-bottom: 0;
+}
+
+.card-separator {
+  margin-left: -24px; /* equals to antd's default card padding */
+  margin-right: -24px; /* as above */
+  margin-bottom: 2vh;
+  border-bottom: 1px solid #ccc;
+}
+
+#card-img {
+  margin: auto;
+  margin-bottom: 2vh;
+  width: 18vh;
+  height: 18vh;
+  border-radius: 50%;
+}
+
+#card-name {
+  font-size: 3.5vh;
+  margin-bottom: 0;
+}
+
+#card-semester {
+  font-size: 2.5vh;
+}
+
+#card-about {
+  text-align: right;
+  font-size: 1.9vh;
+}
+
+#card-work-division, .card-time {
+  margin: 0;
+  font-size: 2vh;
+}
+
+.card-small-text {
+  font-size: 1.5vh;
+}
+
+.rating-card::-webkit-scrollbar {
```

Comment:
What is this?

Reply:
Some basic css for the scrollbar, instead of the regular html one:
![image](https://user-images.githubusercontent.com/35609587/77853725-59256700-71ee-11ea-9bce-93a9e8bd22f2.png)
Do you prefer removing it until we decide on a uniform design for the whole project?

Reply:
Oh great - changing the scroll bar is really good.
A few tips on this:
1) Every change you make that you thing could also be good in other places in our code should be in a more general place, and affect all such types. In this example, I think you should move the scrollbar changes to `index.css`, and make them affect all the scrollbars in our website.
2) You could really upgrade the look of the scrollbar by adding the `border-radius` property to the css (maybe with 10px, or you may choose whatever looks nice to you). Also make it a bit brighter and maybe thinner - about 8px wide.

Reply:
Great ideas! it does looks nicer